.mex/setup.sh

```
@@ -0,0 +1,603 @@
+elif command -v node &>/dev/null; then
+  if [ -f "$SCRIPT_DIR/dist/cli.js" ]; then
+    MEX_CMD="node $SCRIPT_DIR/dist/cli.js"
+    ok "CLI engine ready"
+  elif [ -f "$SCRIPT_DIR/package.json" ]; then
+    info "Building mex CLI engine (first-time setup)..."
+    BUILD_LOG=$(cd "$SCRIPT_DIR" && npm install 2>&1) || {
+      warn "npm install failed — continuing without CLI"
+      warn "Run manually: cd .mex && npm install && npm run build"
+    }
+    if [ -d "$SCRIPT_DIR/node_modules" ]; then
+      BUILD_LOG=$(cd "$SCRIPT_DIR" && npm run build 2>&1) || {
```

Shellcheck (0.11.0)

[warning] BUILD_LOG appears unused. Verify use (or export if used externally).

(SC2034)